nextflow_schema.json

```diff
@@ -81,6 +81,13 @@
                     "description": "Alternative to matrix: a compressed CEL files archive such as often found in GEO",
                     "fa_icon": "fas fa-file-archive",
                     "help_text": "Use this option to provide a raw archive of CEL files from Affymetrix arrays. Will be ignored if a matrix is specified."
+                },
+                "querygse": {
+                    "type": "string",
+                    "default": "None",
+                    "description": "Alternative to CEL archive: the GSE identifier as found in GEO",
```

Comment:
Can I contest this one? There's no mention of Affymetrix, just an alternative to CELs which can also apply to Affymetrix?

Reply:
Imagine we add other modalities such as the special non-CEL array formats from Illumina. We'd need to remember to come back and fix this. It's easier if you're just specific about what you mean from the start.

Reply:
(and AFAIK CEL === affy)

Reply:
I meant user may want to use the GSE identifier for an Affy study and use SOFT instead of CEL in that case as well. Although I get your point, and amended it to "Use SOFT files from GEO by providing the GSE study identifier"